src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,31 @@
-export const PostInfo = () => <>Put the post here</>;
+import { CommentList } from '../CommentList/CommentList';
+import { UserInfo } from '../UserInfo/UserInfo';
+
+export const PostInfo = ({ post }) => {
+  const { title, body, comments, user } = post;
+
+  return (
+    <>
+      <div className="PostInfo">
+        <div className="PostInfo__header">
+          <h3 className="PostInfo__title">{title}</h3>
+
+          <p>
+            {' Posted by  '}
+            <UserInfo user={user} />
```

Comment:
user can be undefined, so let's render the check only when user exists
```suggestion
            {user && <UserInfo user={user} />}
```